cookiecutter/main.py

```diff
@@ -50,7 +50,8 @@ def cookiecutter(
     :param replay: Do not prompt for input, instead read from saved json. If
         ``True`` read from the ``replay_dir``.
         if it exists
-    :param output_dir: Where to output the generated project dir into.
+    :param overwrite_if_exists: Overwrite the contents of the output directory if it exists.
```

Comment:
lint fails, line too long. this should solve it.
```suggestion
    :param overwrite_if_exists: Overwrite the contents of the output 
           directory if it exists.
```